lib/public/utilities/formatTimestamp.js

```diff
@@ -43,3 +43,21 @@ export const formatTimestamp = (timestamp, inline = true) => {
         date.toLocaleTimeString(LOCALE_FORMAT, { timeStyle: TIME_STYLE }),
     ]);
 };
+
+/**
+ * Returns a textual value by the use of a boolean or a null value.
+ *
+ * @param {boolean|null} boolean The given boolean
+ *
+ * @return {string} If present it returs on or off, if not it returns a dash.
+ *
+ */
+export const formatBoolean = (boolean) => {
+    if (boolean != null) {
+        if (boolean) {
+            return 'On';
+        }
+        return 'Off';
+    }
+    return '-';
+};
```

Comment:
Can you please put this in its own file?
I think in general the best is one exported function => one file, with the same name as the exported function

Reply:
Done!